packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -94,135 +126,189 @@ const CertificationQuiz: React.FunctionComponent = () => {
           color: 'primary' as ButtonProps['color'],
         }
       : undefined
-
   return (
-    <div className="bootstrap-4-backport CertificationQuiz">
-      {passingRecord && (
-        <div>
-          <>
-            {!passingRecord.passed && (
-              <div className="failBanner">Quiz Failed</div>
-            )}
-            <Typography variant="hintText">
-              Score: {passingRecord.score} / {quiz?.questions.length}
-            </Typography>
-            {passingRecord.passed ? (
-              displayToast(
-                `You passed the Synapse Certification Quiz on ${passingRecord.passedOn}`,
-                'success',
-              )
-            ) : (
-              <Typography variant="body1">
-                Please review the items shown in red below, and{' '}
+    <div>
+      {passingRecord && !isTakingQuiz && (
+        <>
+          {(!passingRecord.passed || !isCertified) && (
+            <Alert severity="error">
+              {!passingRecord.passed && <AlertTitle>Quiz Failed</AlertTitle>}
```

Comment:
Notice the Quiz Failed title is only shown if passing record indicates user failed.  In the case of revocation, this will not be shown.